configure.ac

```diff
@@ -276,6 +276,15 @@ AM_CONDITIONAL(USE_GPGME, test "x$have_gpgme" = xyes)
 
 # These are needed by libcomposefs
 AC_MSG_CHECKING([for new mount API (fsconfig)])
+AC_COMPILE_IFELSE(
+	[AC_LANG_SOURCE([[
+			#include <sys/mount.h>
+      int foo = MOUNT_ATTR_IDMAP;
+	]])],
+	[AC_MSG_RESULT(yes)
+	 have_mount_attr_idmap=yes],
+	[AC_MSG_RESULT(no)])
+AC_MSG_CHECKING([for new mount API (fsconfig)])
```

Comment:
I think you want to use a different message here. Maybe `for new mount API (idmap)`?

Reply:
Thanks, fixed